app.rb

```diff
@@ -1,8 +1,59 @@
 require 'sinatra/base'
+require './lib/player'
+require './lib/game'
+
```

Comment:
I really like the way this is all laid out and how you switch from page to page! It makes a lot of sense when reading it through and has actually helped with my understanding of it (so thank you). But all in all, I think it looks really good :D